Fabric.Identity.API/Fabric.Identity.API.csproj

```diff
@@ -32,6 +32,10 @@
     <None Remove="Persistence\SqlServer\Entities\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Remove="appsettings.Development.json" />
```

Comment:
is this file described somewhere?  Is that the documentation you are creating?

Reply:
this is a copy file of the appsettings.json but it take precedence.   so you can have bogus values in appsettings and then create this file locally with your real secrets.

Reply:
ok